src/@next/Combobox/components/OptionList/OptionListStyle.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../../../utilities/borderRadius';
+import { Blue, Neutral } from '../../../utilities/colors';
+import { space4, space8 } from '../../../utilities/spacing';
+
+interface OptionListContainerProps {
+  textInputWidth?: number;
+}
+
+export const OptionListContainer = styled.div<OptionListContainerProps>`
+  width: ${props => props.textInputWidth}px;
+  padding: ${space8} 0;
+`;
+export const StyledOptionList = styled.ul`
+  list-style: none;
+  margin: 0 ${space8};
+  padding: 0;
+
+  li {
+    display: flex;
+    height: 44px;
+    position: relative;
+    align-items: center;
+    padding: 0 ${space8};
+    margin-bottom: ${space4};
+
+    &:hover {
+      background: ${Neutral.B99};
+      border-radius: ${borderRadius4};
+      cursor: pointer;
+    }
+
+    &[data-active='true'] {
+      background: ${Blue.S08};
```

Comment:
Design change: bg color remains unchange in the active state and thus should be removed.